cloudbuild.yaml

```
@@ -7,7 +7,7 @@ timeout: 1200s
 options:
   substitution_option: ALLOW_LOOSE
 steps:
-  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20211118-2f2d816b90'
+  - name: gcr.io/cloud-builders/docker
```

I think it probably makes sense to switch to this base image since it appears to be update more frequently. With that said, I think we should probably pin to a version. This is the latest release:

```suggestion
  - name: gcr.io/cloud-builders/docker:20.10.14
```

In case we need to fall back to the k8s one, I'd recommend using the latest released version of that (from Oct 7). As far as I can tell, the main reason to use the k8s one is that it includes go, but with the way our Dockerfile is structured, I don't think that helps us:

```
gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20221007-69e0da97ef
```